CMakeLists.txt

```diff
@@ -42,22 +42,36 @@ MESSAGE("-- Dependency Include Hints (DEPS_INCLUDE_HINTS): ${DEPS_INCLUDE_HINTS}
 #
 
 # Find HDF5
-FIND_PACKAGE(HDF5 REQUIRED COMPONENTS HL)
+MESSAGE("-- HDF5 Root: ${HDF5_ROOT}")
+if(HDF5_ROOT)
+  FIND_PACKAGE(HDF5 REQUIRED COMPONENTS HL)
```

Comment:
This fails unless I remove the `COMPONENTS HL` part. Perhaps it's because I have an old HDF5 (1.8.13)? That's the version we normally use for DAGMC and I haven't had a problem using it with pyne in the past.